source/validation.txt

```diff
@@ -22,60 +22,140 @@ Document Validation
    :ref:`MongoDB Compass Readonly Edition <compass-readonly>`.
    In Readonly Edition, validation rules may only be read.
 
-
 .. _validation-tab:
-.. _create-validation-rules:
 
 Validation Tab
 --------------
 
```

Comment:
I'm curious why this page is titled "Document Validation" rather than "Schema Validation", as it is in the server manual. Either is OK, I think, but it might be better to stay consistent.

Also, it would be good to include a paragraph at the top of the page with a short explanation of what schema validation is, i.e. a mechanism for ensuring that all the documents in a collection conform to a certain shape or a certain range of values. The server manual page doesn't have this either, and I think it would be helpful.